app/assets/javascripts/base.js.erb

```diff
@@ -123,15 +123,10 @@ var instantClick
     prog.classList.remove("showing");
 
     if (newUrl) {
-      // We skip showing the "Skip to content" button for the search page, see
-      // https://github.com/forem/forem/issues/13876
-      var path = new URL(newUrl, window.location.origin).pathname;
-      if (path != '/search') {
-        const skipLink = document.querySelector('.skip-content-link');
+      const skipLink = document.querySelector('.skip-content-link');
         if (skipLink) {
           skipLink.focus();
         }
```

Comment:
Refactor (blocking): Prefer optional chaining. Currently, there isn’t a lint rule for optional chaining.

```suggestion
        document.querySelector('.skip-content-link')?.focus();
```

Reply:
We can't use it in a `js.erb` file it seems 🙁  From the buildkite build:

```
Uglifier::Error: Unexpected token: punc (.)
```

Reply:
That's correct, Uglifier does not support optional chaining. The last release is from 2019, most likely nothing newer than that will work.

Reply:
Oops, I missed that this was in an erb. In that case all good. Don't know if it's worth it here, but we could potentially bring this into an ES module/pack file in a separate PR.